Yoast/wordpress-seo
# Indexable home page watcher

## Summary

Creates a home page watcher class that watches updates to any options that affect the home page indexable and updates the home page indexable accordingly.

## Test instructions

This PR can be tested by following these steps:

* Make sure your front page is set to the posts page.
* Visit your home page and check that current presenters give expected output.
* Under SEO -> Search Appearance update settings for your home page.
* Check that the presenters update their output accordingly.

## Documentation

## Quality assurance
